app/src/main/java/com/stevesoltys/seedvault/transport/backup/PackageService.kt

```diff
@@ -28,13 +28,13 @@ private const val LOG_MAX_PACKAGES = 100
  */
 internal class PackageService(
     private val context: Context,
-    private val backupManager: IBackupManager,
+    private val settingsManager: SettingsManager,
 ) {
 
     private val packageManager: PackageManager = context.packageManager
     private val myUserId = UserHandle.myUserId()
 
-    val eligiblePackages: Array<String>
+    val requestedPackages: Array<String>
```

Comment:
Would be nice to add a Kotlin do to this field as the name isn't immediately clear to me. 

It seems to be only used when manually requesting a backup, so I wonder if backups by the system are actually working as D2D now as well. Did you try this?